docs/companion_guide.md

```diff
@@ -322,8 +322,34 @@ a message back.
 
 ### Q&A
 
+#### General Questions
+
+- **How do you pronounce DWNs?**: We've heard a few ways to say it:
+  - DAWN
+  - D Web Node
+  - D W N
+- **How are DWNs different than SOLID Pods?**
 - **For the base case many DWNs should I expect a particular person to have?** A person is
   expected to have a few DWNs. Maybe more than 1 but less than 10.
+
+#### Security Questions
+
+- **Are there Data Privacy Considerations like GDPR. And how are they taken care of in this kinds of a paradigm?**
+  DWN's are basically GRPR by default as you are 100% in control of your data store.
+- **What is the best way to ensure that recipients of PII access via DWN are not
+  persistently storing the information using their own digital agent?**
+  The in-browser embedded DWN does have storage restrictions. Let’s say I have a
+  second DWN hosted by some provider, can the second DWN use a different store,
+
+#### Specification Questions
+
+- **How flexible/dynamic are the protocol control rules?** The point of
+  protocols is so you can create a protocol. What type of actors can do what. Not
```

Comment:
"The point of protocols is so you can create a protocol."

The intended meaning of this circular sentence is indecipherable.

Reply:
@TallTed absolutely. To give myself some clearance here, these were notes I took on the fly during the AMA call. I won't be moving out of draft until I've cleaned up some of these questions and gotten them more consumable for everyone else. It's mostly a PR for visibility, as we discussed this over the DWN call last week. That being said, I very much appreciate the review and encourage you to continue to review the PRs that come forward. I'll probably make a cleanup round by the next DWN call (next week) where I'll move out of draft into an actual PR. 

Thanks!